rs/bitcoin/checker/tests/tests.rs

```diff
@@ -373,6 +375,10 @@ fn test_check_transaction_passed() {
         let actual_cost = cycles_before - cycles_after;
         assert!(actual_cost > expected_cost);
         assert!(actual_cost - expected_cost < UNIVERSAL_CANISTER_CYCLE_MARGIN);
+        // Check metrics
```

Comment:
nit: here and below is this comment needed? I would argue it's self explanatory given the name `MetricsAssert`, but no strong feeling.